docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
```

Comment:
The volume mapping for media files is correctly set up. This ensures that media files are persisted outside of the container, which is a good practice for handling media in Docker.